server2/Arcadia.Assistant/Arcadia.Assistant.ExternalStorages.SharepointOnline/Contracts/ISharepointFieldsMapper.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Arcadia.Assistant.ExternalStorages.SharepointOnline.Contracts
+{
+    using System;
+    using System.Linq.Expressions;
+
+    using Abstractions;
+
+    public interface ISharepointFieldsMapper
+    {
+        string GetSharepointField(Expression<Func<StorageItem, object>> property);
+
+        string? GetSharepointField(string propertyName);
```

Comment:
Why did you do this a part of public interface if it is not used anywhere except internal implementation?

Reply:
It used externally before. Fixed.